app/helpers/application_helper.rb

```diff
@@ -16,7 +16,7 @@ def page_entries_info(collection)
   def execute_actions(resource, class_name)
     api_actions = resource.send(class_name)
     api_actions.each do |api_action|
-      api_action.execute_action unless api_action.serve_file? || api_action.redirect?
+      api_action.execute_action unless api_action.serve_file? || api_action.redirect? || api_action.custom_action?
```

Comment:
we should document that in the order of operations, custom actions should come last! @alis-khadka

Reply:
I did not understand it clearly